README.md

```
@@ -2,6 +2,10 @@
 
 The official website for the IT Committee of Koishikawa Secondary School (都立小石川中等教育学校).
 
+## 日本語版はこちら
+
+[リンク](./README-ja.md)
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)